File: ViewModelsLayer/ViewModels/MoveViewModels/ResponseMoveViewModel.cs
using ViewModelsLayer.ViewModels.CardViewModels;
using ViewModelsLayer.ViewModels.UserViewModels;

namespace ViewModelsLayer.ViewModels.MoveViewModels
{
    public class ResponseMoveViewModel
    {
        public ResponseUserViewModel User { get; set; }
        public ResponseCardViewModel Card { get; set; }
    }
}

File: API/Angular/src/app/viewmodels/MoveViewModels/response-move-view-model.ts
import { ResponseUserViewModel } from '../UserViewModels/response-user-view-model';
import { ResponseCardViewModel } from '../CardViewModels/response-card-view-model';

export default class ResponseMoveViewModel {
  constructor(user, card) {
    this.card = card;
    this.user = user;
  }

  public user: ResponseUserViewModel;
  public card: ResponseCardViewModel;
}
